src/styles/variables.scss

```diff
@@ -0,0 +1,10 @@
+$stopwatch-size: 80vmin;
+$minutes-hand-height: 20vmin;
+$minutes-hand-width: 3vmin;
+$seconds-hand-heigth: 38vmin;
```

Comment:
There is a typo in the variable name `$seconds-hand-heigth`. It should be `$seconds-hand-height` to maintain consistency and avoid errors in other files where this variable is used.